src/python/tests/unittests/test_lftp/test_job_status_parser.py

```
@@ -160,6 +160,37 @@ def test_jobs_v_echo_on_own_line_in_chunk(self):
+    def test_chunk_without_data_line(self):
+        """Test that a \\chunk line at end of output (no data line) doesn't crash."""
+        output = (
+            "jobs -v\n"
+            "[2] pget -c ~/downloads/completed/SomeFile.mkv -o /data/lftpsync// \n"
+            "    sftp://user:pass@host:22/home/user\n"
+            "    `~/downloads/completed/SomeFile.mkv', got 1000 of 2000 (50%) \n"
+            "  \\chunk 0-999\n"
+        )
+        parser = LftpJobStatusParser()
+        statuses = parser.parse(output)
+        self.assertEqual(1, len(statuses))
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)